seeds/seed_data.sql

```diff
@@ -125,37 +127,39 @@ INSERT INTO "incomes" ("id", "owner_id", "category_id", "description", "amount",
         8,
         'Direct Deposit',
         250500.00,
+        make_timestamptz(2019, 1, 1, 0, 0, 0),
         null
     );
 
 ALTER SEQUENCE IF EXISTS incomes_id_seq RESTART WITH 4;
 
-INSERT INTO "expenses" ("id", "owner_id", "category_id", "description", "amount", "recurring_rule")
+INSERT INTO "expenses" ("id", "owner_id", "category_id", "description", "amount", "start_date", "recurring_rule")
   VALUES
     (
         1,
         1,
         1,
         'Rent',
         1200.00,
-        'DTSTART:20181001T000000Z
-RRULE:FREQ=MONTHLY;INTERVAL=1;WKST=MO'
+        make_timestamptz(2019, 1, 1, 0, 0, 0),
```

Comment:
What do these changes do?

Reply:
Breaking the recurring rule and start dat out into own columns